release/release_test.go

```diff
@@ -62,6 +66,30 @@ BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
 	return mockOSRelease
 }
 
+// Kernel magic numbers
+const (
+	wslfs int64 = 0x53464846
+	ext4  int64 = 0xef53
+)
```

Comment:
nitpick: thse could be defined in release.go and then exported to the test via export_test.go

Reply:
I prefered to keep it in the tests to prevent them from being exported in the future. I figured the test module was as private as it gets.

I wanted this because:
1. `wslfs` is not the result of a real kernel call but rather Microsoft's WSL1 emulation: You will find neither [upstream](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/magic.h?h=v6.0) nor in [wsl2's](https://github.com/microsoft/WSL2-Linux-Kernel/blob/7a924ef48aeef0fc5d835548b4a202c3888590a1/include/uapi/linux/magic.h) list of magic numbers.
3. The list is far from exhaustive.